lib/expand.js

```diff
@@ -61,20 +61,6 @@ class Expander {
       }
     }
 
-    const entryDE = this._expanded.dataElements.find('shr.base', 'Entry');
-    let entryFields = null;
-    if (!entryDE) {
-      // 12036, 'Could not find expanded definition of ${element}. Inheritance calculations will be incomplete.', 'Double check shr.base.Entry is defined within the specifications.', 'errorNumber'
-      logger.warn({ element: 'shr.base.Entry' }, '12036');
```

Comment:
I thought about that _once_ -- and then promptly forgot about it. ;-)

I think we should delete them.  When I first thought of it, I wondered if we should keep it for historical purposes -- but on second thought, that's silly -- because that's what we have Git for.  I'll delete the messages.

Reply:
Fixed in 12ec1b1.